### PR TITLE
Logout

### PR DESCRIPTION
**Pull request by Shafizadegan**
See active in NavLinks
See Page Logout

### CODE REVIEW

File: front/se_project/src/components/logout.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React, {Component} from 'react';
```
Comment (Z-Ramezani) [inline front/se_project/src/components/logout.jsx:1]: check does it works with back?
check with cookie

File: front/se_project/src/components/navBar/navLinks.css [hunk 2]
```diff
@@ -4,9 +4,12 @@
 }
```
Comment (Z-Ramezani) [inline front/se_project/src/components/navBar/navLinks.css:4]: checked!

**Review by Z-Ramezani — CHANGES_REQUESTED** [review]
check comments before merge

File: front/se_project/src/components/logout.jsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+//     componentDidMount() { 
+//         localStorage.removeItem('token');
+//         window.location = '/'
+//     }
+
+//     render () {
+//         return null;
+//     }
+// }
+// export default Logout;
```
Comment (Madanian) [inline front/se_project/src/components/logout.jsx:14]: that's ok. we should test it after merging!

**Review by Madanian — CHANGES_REQUESTED** [review]
checked

File: front/se_project/src/components/logout.jsx (hunk 1 above)
Comment (Shafizadegan) [reply front/se_project/src/components/logout.jsx:1]: ok

File: front/se_project/src/components/logout.jsx (hunk 3 above)
Comment (Shafizadegan) [reply front/se_project/src/components/logout.jsx:14]: yes

**Pull request merged**